CHANGELOG.md

```diff
@@ -14,6 +14,8 @@ This is the log of notable changes to EAS CLI and related packages.
 
 ### 🧹 Chores
 
+- Upgrade `@expo/eas-build-job` to `1.0.56`, thus removing support for S3 project archives. ([#2165](https://github.com/expo/eas-cli/pull/2165) by [@sjchmiela](https://github.com/sjchmiela))
```

Comment:
Nit: It sounds scary, even though I know it isn't. Can you add an explanation as to why it is not dangerous/breaking change to the PR description so it stays here for the future?